Sampled/UI/VIews/LibraryView.swift

```diff
@@ -803,15 +872,55 @@ struct LibraryView: View {
       infoTrack.discNumber = tracks.reduce(.empty) { $0.reduce(nextValue: $1.discNumber) }
       infoTrack.discTotal = tracks.reduce(.empty) { $0.reduce(nextValue: $1.discTotal) }
     }
+      }
+      
+      // Combined player - only show when track(s) selected
+      if !selection.isEmpty {
+        Divider()
```

Comment:
I think it would be better if these core elements weren't as interactive. I may give you a license to Doppler and Meta so you can see what I'm up against.